pkg/server/thick_cni_test.go

```diff
@@ -221,10 +221,7 @@ func prepareCNIEnv(netnsPath string, namespaceName string, podName string, podUI
 	if err := os.Setenv("CNI_NETNS", netnsPath); err != nil {
 		return err
 	}
-	if err := os.Setenv("CNI_ARGS", cniArgs); err != nil {
-		return err
-	}
-	return nil
+	return os.Setenv("CNI_ARGS", cniArgs)
```

Comment:
Is CNI_ARGS going to be the new default instead of returning nil? And what is the reason for this change?